api/src/main/resources/messages_en_GB.properties

```diff
@@ -11,6 +11,8 @@ Concept.stats.histogramOutliers=Histogramme (w/o Outliers)
 cohortBuilder.yearOld=a\u00f1os
 
 reporting.noDrugOrders=No drug orders
+general.back=Back
```

Comment:
Do we need to add this when in it already in messages.properties?

Reply:
i guess not